Лаб 1/1.py

```diff
@@ -0,0 +1,19 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+
+numbers1 = numbers[5:] # после неизвестного
+numbers2 = numbers[:4] # до неизвестного
+n = numbers2 + numbers1
+
+l = len(n)
+#print(l, 'len')
+
+s = sum(n)
+#print(s, 'sum')
+
+ar = (s / (l+1))
+#print(n)
+
+numbers[4] = ar
```

Comment:
И вот итоговая оптимизация, когда практически вся задача умещается в одну строку кода: 
```suggestion
numbers[numbers.index(None)] = sum(numbers[:numbers.index(None)] + numbers[numbers.index(None) + 1:]) / len(numbers)
```